LEGO1/act3shark.cpp

```diff
@@ -0,0 +1,4 @@
+#include "act3shark.h"
+
+// 0x100f03a0
+static char* g_act3SharkString = "Act3Shark";
```

Comment:
I'm fairly certain this is not a separate global static var. All `ClassName()` implementations return a string, but the string is stored in the data section of the executable, so the result is it always returns the address of a seemingly static variable.

Reply:
Ahh my bad. In my haste, I missed that comment and didn't realize this was still being worked on.

Still, there are a few PRs semi-waiting for this now, and I'm still strongly considering fixing up the remaining issues myself so it can be merged.

Reply:
Sure, I have a bunch of local changes and merged 4db49fe6a6e3480413b37706c41fed51163f82b0 in. 

Do you want me to finish up what I have currently or bring over the version here?

Reply:
Up to you, if you have stuff you'd like to finish, I think you may as well

Reply:
Updated! If you want I can merge the current master back in, otherwise feel free to take it over and do it yourself.